### ISSUE TITLE
uv_uptime broken on Windows

### ISSUE THREAD

**Issue by saghul** [joyent/libuv#375]
After 442aa1f4696d24a605790b8ad4317696758ee8de it no longer works. I tested it on a Windows 7 (32bit) machine using MinGW.

This is the output I'm getting, though it's not very meaningful :-S When running inside GDB it doesn't report anything either.

```
saghul@win7dev /c/Users/saghul/src/libuv/test
$ ./run-tests.exe platform_output
`platform_output` failed: exit code -1073741819
Output from process `platform_output`:
uv_get_process_title: MINGW32:/c/Users/saghul/src/libuv/test
uv_resident_set_memory: 2052096
=============================================================
```

Reverting that commit fixed it.

**Comment by piscisaureus** [comment]
Works for me (using msvc 2010, Windows 7 enterprise)

**Comment by piscisaureus** [comment]
@saghul Can you try this branch? https://github.com/joyent/libuv/commits/windows-uptime

**Comment by saghul** [comment]
@piscisaureus It works! You're the man! :-)

**Comment by piscisaureus** [comment]
Ok, a slightly modified version of this patch was landed in 9984d157ffcd8ab71fee6a6b6888c08d0f277f83.
